README.md

```diff
@@ -82,14 +86,17 @@ We've turned on [discussions](https://github.com/UCL-ARC/python-tooling/discussi
    ```
 
 5. To work on your project, initialise a git repository and _install_ it in editable mode.
-   (If you want to develop in a `venv` or `conda` environment, now is a good time to set that up)
    ```
    cd PROJECT_SLUG
    git init
-   # conda create -n PROJECT_SLUG-dev python=3.11
-   # conda activate PROJECT_SLUG-dev
    python -m pip install -e .
    ```
+6. Build your package
+   ```
+   python -m build
+   python -m setuptools_scm
```

Comment:
does this need to be run explicitly? I think it might be handled by `build`

Reply:
Agreed, that second line is not needed. `build` uses the backend specified.
```suggestion
```